docs/commands/oc-native-host.md

```
@@ -17,8 +17,40 @@ Options:
+  - Chromium Registry Key: HKEY_CURRENT_USER\Software\Chromium\NativeMessagingHosts\com.calycode.cli
+  - Chromium Registry Configured: Yes
+  - Expected Extension IDs: hadkkdmpcmllbkfopioopcmeapjchpbm, lnhipaeaeiegnlokhokfokndgadkohfe
+  - Expected Origins: chrome-extension://hadkkdmpcmllbkfopioopcmeapjchpbm/, chrome-extension://lnhipaeaeiegnlokhokfokndgadkohfe/
+  - Manifest Allowed Origins: chrome-extension://hadkkdmpcmllbkfopioopcmeapjchpbm/, chrome-extension://lnhipaeaeiegnlokhokfokndgadkohfe/
+```
 
 Run 'caly-xano <command> --help' for detailed usage.
 https://github.com/calycode/xano-tools | https://links.calycode.com/discord
-```
\ No newline at end of file
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)